src/model/song.rs

```diff
@@ -0,0 +1,53 @@
+use crate::Chord;
+use combine::{parser::char::spaces, sep_by, Parser, Stream};
+use std::fmt;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Song {
+    pub chords: Vec<Chord>,
+}
+
+impl Song {
```

Comment:
There must be a better name than this

Reply:
The terms "progression" or "chord progression" come to mind (or "changes"/"chord changes", or "chart"/"chord chart")

Reply:
Switching to `Progression`.